src/Components/Home.js

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { FaGlobe } from 'react-icons/fa';
+import { fetchCountries } from '../redux/slices/countriesSlice';
+
+function Home() {
+  const { countries } = useSelector((state) => state.countries);
+  const [searchTerm, setSearchTerm] = useState('');
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchCountries());
+  }, [dispatch]);
+
+  const handleSearch = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
```

Comment:
- [ ] The Home section should contain a banner section that gives context to the application as we have in the design. Kindly add the banner section to your app as previously requested.

  Your design (no banner section) | Design
  :---:|:---:
  ![image](https://user-images.githubusercontent.com/54848363/220663696-87f2cdf0-d784-4dca-8417-0063e93d59ea.png) | ![akumu-descriptive6](https://user-images.githubusercontent.com/54848363/220663509-c0970cf4-1d22-4820-971f-4bcb7698d2f3.jpg)

Reply:
@DammyShittu I did not get what you mean by that banner.